test/integration/setting_js_test.rb

```diff
@@ -29,4 +29,78 @@ class SettingJSTest < IntegrationTestWithJavascript
     assert_index_page(settings_path, "Settings", false, true, false)
     assert page.has_link?("My Pretty Setting Label", :href => "#category_label_test_settings_tab")
   end
+
+  test "general headers values" do
+    visit settings_path
+
+    expect page.has_content?('Name')
+    expect page.has_content?('Value')
+    expect page.has_content?('Description')
+  end
+
+  test "edit single arrays values" do
+    visit settings_path
+    find('button#http_proxy_except_list').click
+
+    input = find('textarea#setting-textarea-http_proxy_except_list', visible: true)
+    expect input.value.empty?
```

Comment:
I think the more specific matcher `assert_empty` will give a better result when it doesn't match.
```suggestion
    assert_empty input.value
```